static/js/chart/draw.ts

```diff
@@ -213,6 +213,68 @@ function drawSingleBarChart(
     .attr("fill", (d) => color(d.label));
 }
 
+function ramp(color, n = 256) {
```

Comment:
How do we add a function comment in TypeScript?  It should explain the function's purpose, the arguments, and the returned value.

Reply:
Added based on other comments in this file as well as `choropleth.tsx`.